src/com/phraseapp/androidstudio/API.java

```diff
@@ -27,14 +27,53 @@ public API(String clientPath, String access_token, Project project) {
 
     // Get all locales
     public APIResourceListModel getLocales(String projectId) {
-        List<String> params = new ArrayList<String>();
+        int page = 0;
+        APIResourceListModel locales = getLocalePage(projectId, page);
+        while (!locales.isEmpty()) {
```

Comment:
das duerfte so nicht funktionieren. falls `locales.isEmpty == false` wird sich diese bedingung doch nie aendern in der loop. oder uebersehe ich da was?